recipes/crispector/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: crispector
+  version: "1.0.0"
+
+build:
+  number: 0
+  entry_points:
+    - crispector = crispector.arg_parser:cli
+  noarch: generic
+
+source:
+  url: https://github.com/YakhiniGroup/crispector/archive/v1.0.0b.tar.gz
+  sha256: 3dcdad7b4a4dd317a1137186be98376b367ebb20ce7e56e0bb58e1469a04bcde
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools
+    - biopython 1.74
```

Comment:
This dependency and the following ones should be removed from the `host` dependencies. They are not required at install time when using `pip install --no-deps`. But you should add `pip` as a host dependency.

Reply:
Should I remove all "host:" dependencies and keep just "run:" section?

Reply:
No, you need to keep `python`, `setuptools` and `pip`.